### PR TITLE
Ledger wallet version 1.0.0

### PR DESCRIPTION
**Pull request by Faizal-aelf**
Ledger wallet version 1.0 include following item below:
- QA bug fixes.
- UI enhancement.
- Updated code to support only mainnet and main chain alone.
- Hardcoded mainnet wallet address.

### CODE REVIEW

File: components/common/header/index.tsx [hunk 1]
```diff
@@ -29,9 +27,6 @@ const Header = ({
             <Image src={network == NetworkStateEnum.testnet ? logoTestnetImage : logoMainnetImage} alt="Aelf logo" height={36}/>
```
Comment (yongenaelf) [inline components/common/header/index.tsx]: "AELF" or "aelf", don't use "Aelf"

File: components/transactions/components/SendTransaction.tsx [hunk 2]
```diff
@@ -261,6 +266,14 @@ function SendTransaction({
                   stringMode
                   parser={(value) => value.replace(/[\s$,]/g, "")}
                   onChange={setAmountValue}
+                  onStep={(value, stepObj: any) => {
```
Comment (yongenaelf) [inline components/transactions/components/SendTransaction.tsx]: avoid using the "any" type

File: components/transactions/components/TransferVerification.tsx [hunk 3]
```diff
@@ -69,8 +69,12 @@ const TransferVerification = ({
       );
       const {Success, TransactionFee } = await feeResponse.json();
       if (Success) {
-        const calculatedFees = Number(new BigNumber(TransactionFee.ELF).dividedBy(10 ** 8).toNumber());
-        setFees(calculatedFees);
+        if (isEmptyObject(TransactionFee)) {
+          setFees('-');
+        } else {
+          const calculatedFees = Number(new BigNumber(TransactionFee.ELF).dividedBy(10 ** 8).toNumber());
```
Comment (yongenaelf) [inline components/transactions/components/TransferVerification.tsx]: typeof new BigNumber(0).toNumber() is already number, no need to cast to Number

File: components/transactions/constants.ts [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
   [ChainStateEnum.tDVW]: 'SideChain '.concat(ChainStateEnum.tDVW),
+}
+
+export const getLastTwoUnits = (date) => {
+  const now = new Date();
+  const distance = differenceInMilliseconds(now, date);
+
+  const days = Math.floor(distance / (1000 * 60 * 60 * 24));
+  const hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+  const minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
```
Comment (yongenaelf) [inline components/transactions/constants.ts]: We can use library.


```ts
import { intervalToDuration, formatDuration } from "date-fns";
export const dateToWords = (date: Date) => {
     const duration = intervalToDuration({ start: date, end: new Date() });
     return formatDuration(duration).replace("hour", "hr").replace("minute", "min") + " ago";
}
```

File: hooks/useMultiTokenContract.ts [hunk 5]
```diff
@@ -8,7 +9,7 @@ export const useMultiTokenContract = () => {
   const aelf = useAElf();
 
   return useSWR([aelf, "multiTokenContract"], async () => {
-    const tokenContractName = "AElf.ContractNames.Token";
+    /* const tokenContractName = "AElf.ContractNames.Token";
```
Comment (yongenaelf) [inline hooks/useMultiTokenContract.ts]: delete this comment, we can always restore it later using git history

**Review by yongenaelf — CHANGES_REQUESTED** [review]

File: components/common/header/index.tsx (hunk 1 above)
Comment (Faizal-aelf) [reply components/common/header/index.tsx]: Code updated.

File: components/transactions/components/SendTransaction.tsx (hunk 2 above)
Comment (Faizal-aelf) [reply components/transactions/components/SendTransaction.tsx]: Code updated.

File: components/transactions/components/TransferVerification.tsx (hunk 3 above)
Comment (Faizal-aelf) [reply components/transactions/components/TransferVerification.tsx]: Code updated.

File: components/transactions/constants.ts (hunk 4 above)
Comment (Faizal-aelf) [reply components/transactions/constants.ts]: Code updated.

File: hooks/useMultiTokenContract.ts (hunk 5 above)
Comment (Faizal-aelf) [reply hooks/useMultiTokenContract.ts]: Code updated.

**Review by yongenaelf — APPROVED** [review]

**Pull request merged**